andela/travel_tool_front
# #159392536 Implement requests page header

#### What does this PR do?
This PR Implements requests page header (UI Only)


#### Description of Task to be completed?
 - implement the page header of the requests page
 - implement the app logo, search field, notifications icon and account dropdown.
 - Write out tests for the components

#### How should this be manually tested?
- Clone the repository
- Install all dependencies using `yarn install`
- Run `./docker.sh development` docker command
- Start the server using `yarn start`
- On the browser, you should be able to see the page displayed with the navigation bar.

#### Any background context you want to provide?
None


#### What are the relevant pivotal tracker stories?
[#159392536](https://www.pivotaltracker.com/story/show/159392536)

#### Screenshots (if appropriate)

 The Requests page header


<img width="1280" alt="screen shot 2018-08-07 at 18 45 00" src="https://user-images.githubusercontent.com/12033494/43786942-84b6b324-9a72-11e8-9e2a-9ddc95298818.png">


 The Requests page navigation bar

<img width="1280" alt="screen shot 2018-08-07 at 18 45 10" src="https://user-images.githubusercontent.com/12033494/43786969-964dc83e-9a72-11e8-811a-10350b9b516f.png">





#### Questions:
None
